### PR TITLE
Add Zap icon

### PR DESCRIPTION
**Pull request by jumasheff**

### CODE REVIEW

File: components/Icons.es6.js [hunk 1]
```diff
@@ -1837,6 +1837,14 @@ function Spaceship ({ className, width="16", height="16", viewBox="0 0 16 16", c
   )
 }
 
+function Zap ({ className, width='11', height='16', color="#D7D7DE", viewBox="0 0 11 16" }) {
```
Comment (adbharadwaj) [inline components/Icons.es6.js:1840]: I am curious why is the width 11 and not 16 like other icons?

Comment (adbharadwaj) [reply components/Icons.es6.js:1840]: Do you have screenshots to compare how both differ?

File: README.md [hunk 2]
```diff
@@ -39,14 +39,14 @@ If you are developing on a remote machine and want to view the demo in your brow
 * Then you can access the demo interface at `http://your-remote-server.com:{port}`.
 
 ## How can I test my changes in an internal repository without publishing?
+#### Note: If you're using docker, then you should do this outside of docker.
```
Comment (adbharadwaj) [inline README.md:42]: Changes look good, but just curious why you made these changes.

File: components/Icons.es6.js [hunk 3]
```diff
@@ -1837,6 +1837,14 @@ function Spaceship ({ className, width="16", height="16", viewBox="0 0 16 16", c
   )
 }
```
Comment (adbharadwaj) [inline components/Icons.es6.js:1838]: In the past, we have made 3 file changes for icons. Why only 1 in this case?

File: README.md (hunk 2 above)
Comment (jumasheff) [reply README.md:42]: I tried to follow these steps, but they were incorrect and misleading. That's why I thought it's was a good time to introduces the fixes.

File: components/Icons.es6.js (hunk 1 above)
Comment (jumasheff) [reply components/Icons.es6.js:1840]: 11 to 16 requirement is defined in the source of the icon (see below)

File: components/Icons.es6.js (hunk 3 above)
Comment (jumasheff) [reply components/Icons.es6.js:1838]: Add the icon to the demo component 😄

**Review by adbharadwaj — APPROVED** [review]
a!

**Pull request merged**